internal/resolve/gosource/gosource.go

```diff
@@ -237,6 +237,11 @@ func withoutStdblibPackages(result *[]string, env *goEnv, paths []string) error
 		if err != nil {
 			return err
 		}
+
+		if strings.Contains(abs, env.GoCache) {
```

Comment:
Isn't this condition too weak? If cache is `/c', for example, this code will ignore `/my/code`.

Reply:
yes, thanks
I should check for `strings.HasPrefix`

Reply:
fixed